source/macos/AdaptiveCards/AdaptiveCards/Views/ACRView.swift

```diff
@@ -75,8 +77,14 @@ class ACRView: ACRColumnView {
     }
     
     func resetKeyboardFocus() {
-        guard isLayoutDoneOnShowCard, let lastFocusedElement = currentFocusedActionElement else { return }
-        lastFocusedElement.controlView?.setAccessibilityFocused(true)
+        if isLayoutDoneOnShowCard, let lastFocusedElement = currentFocusedActionElement {
+            lastFocusedElement.controlView?.setAccessibilityFocused(true)
+        } else if ACRView.focusedElementOnHideError != nil {
+            ACRView.focusedElementOnHideError?.setAccessibilityFocused(true)
+            // If it is a textfield, the entered text gets selected when focus is set, so taking the cursor to the last position it was present
+            guard let focusedTextField = ACRView.focusedElementOnHideError as? ACRTextField else { return }
+            focusedTextField.currentEditor()?.selectedRange = NSRange(location: focusedTextField.lastKnownCursorPosition ?? focusedTextField.stringValue.count, length: 0)
```

Comment:
Won't this occur too for multiline field?

Reply:
That has a ACRTextView, which works out of the box